### PR TITLE
Allow empty values placeholders

### PR DESCRIPTION
**Pull request by vaseker**
Rebased to develop branch

### CODE REVIEW

File: lib/LocalizationProvider.js [hunk 1]
```diff
@@ -91,6 +92,13 @@ function LocalizationProvider($localizationLoader, l10n) {
 LocalizationProvider.prototype._cookieConfig = null;
 
 /**
+ * Return localization key in case of absent value.
```
Comment (pragmadash) [inline lib/LocalizationProvider.js]: I think it's better "Current state of returning placeholder if a value is not found.". Now it seems confusing like this is a method.

Comment (vaseker) [inline lib/LocalizationProvider.js]: Fine.
Can I fix that using pushforce?

Comment (pragmadash) [inline lib/LocalizationProvider.js]: I don't mind.

**Comment by vaseker** [comment]
UP!

**Comment by pragmadash** [comment]
:+1:

**Pull request merged**